### DISCUSSION TITLE
New split window with specific height

### DISCUSSION THREAD

**Discussion by dalvand** [kovidgoyal/kitty#7180] [General]
I use this command to open up a split in kitty window. 

`kitty @ --to unix:/tmp/mykitty_vifm launch --cwd=current --type=window --location=hsplit`

It works. It opens up a split in the bottom half of the kitty window. 

I was wondering if there is any way to specify the height of the split, e.g. to one third of the screen or something less than half?

Thank you.

**Comment by kovidgoyal** [comment]
You can resize the window after it is created, see kitten @ resize-window -h

**Reply by dalvand** [reply]
Could you elaborate on how I can use it? Should I combine it with my command above? Or, should I get the window ID and then pass it to the kitten command you shared? I couldn't find much information or any example online. Thank you.
